modules/sand/CHANGELOG.md

```diff
@@ -8,3 +8,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 - Upgraded to Katamari 8.0, which is incompatible with Katamari 7.0 if used in the same bundle.
+- Detection logic has been updated to use `navigator.userAgentData` for browser detection if it is available.
```

Comment:
This maybe should be an improvement? Not sure, what do others think?

Reply:
Yep, happy to move it